ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -153,11 +164,16 @@ private[apiserver] final class ApiUserManagementService(
         for {
           userUpdate <- handleUpdatePathResult(user.id, UserUpdateMapper.toUpdate(user, fieldMask))
           _ <- identityProviderExistsOrError(user.identityProviderId)
+          authorizedUserContext <- authorizedUserContextFO
+          _ <- verifyPartyExistInIdp(
+            Set(),
+            user.identityProviderId,
+            authorizedUserContext.isParticipantAdmin,
+          )
           _ <- verifyUserIsNonExistentOrInIdp(user.identityProviderId, user.id)
```

Comment:
This is confusing: a user not-existing will result in a success from this method call.
Can you add a comment that this is nevertheless expected to result in an error from the user store layer?